core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -106,6 +106,11 @@ public TriviaButtonBuilder() {
       questionButton.invalidate();
     yButtonPosition -= questionButton.getHeight()/3;
     correctAnswerButton = new TextButton(currentQuestion.correctAnswer, textButtonStyle);
+    correctAnswerButton.getLabelCell().width(300);
+    correctAnswerButton.setWidth(300);
```

Comment:
It seems like 300 represents the button width, and is used as such several times.  Consider making it a constant.